FindLibKrb5.cmake

```diff
@@ -17,11 +17,13 @@
 #  LibKrb5_LIBRARY                 The Krb5 library
 #  LibKrb5_INCLUDE_DIR             The location of Krb5 headers
 
-find_path(LibKrb5_ROOT_DIR NAMES include/krb5.h)
+if (NOT LibKrb5_ROOT_DIR)
+    find_path(LibKrb5_ROOT_DIR NAMES include/krb5.h)
+endif ()
 
 find_library(LibKrb5_LIBRARY NAMES krb5 HINTS ${LibKrb5_ROOT_DIR}/lib)
 
-find_path(LibKrb5_INCLUDE_DIR NAMES krb5.h HINTS ${LibKrb5_ROOT_DIR}/include)
+find_path(LibKrb5_INCLUDE_DIR NAMES krb5/krb5.h HINTS ${LibKrb5_ROOT_DIR}/include)
```

Comment:
Uh... this is confusing. The one include we have in `KRB.h`:
```
src/analyzer/protocol/krb/KRB.h:#include <krb5.h>
```
Should that include be changed to `krb5/krb5.h` now, assuming `LibKrb5_INCLUDE_DIR` ends up with `/usr/include` ?